contracts/test/SetBuilderElf.sol

```diff
@@ -0,0 +1,24 @@
+// Copyright 2024 RISC Zero, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+// This file is automatically generated
+
+pragma solidity ^0.8.20;
+
+library Elf {
+    string public constant SET_BUILDER_PATH =
+        "/Users/victorgraf/risc0/ethereum/target/riscv-guest/guest-set-builder/set-builder/riscv32im-risc0-zkvm-elf/release/set-builder";
```

Comment:
It seems very strange to have your username checked in

Reply:
c7359c8bc23bfe8b0bb5f97a134a22e9b259d1a0